src/ManojN/Q3checkNumberInArray.java

```diff
@@ -0,0 +1,26 @@
+/*
+3) Check if array contains a number in Java?
+Hint: Your method should accept 2 parameters. Input array and number.
+i/p :- arr[] = {2,5,9,7,4,6}
+number :- 8
+o/p :- number is not in arr.
+ */
+package ManojN;
+public class Q3checkNumberInArray {
+	static void m1(int arr1[], int num){
+		//int arr[] = {2,5,9,7,4,6};
+		for(int i=0; i<arr1.length; i++){
+			if (num==arr1[i]){
+			System.out.println("num is same");
+			
+			}	else if (num!=arr1[i]){
+				System.out.println("missing no from array : " + num);
+				break;
+			}
+		}		
+	}
+	public static void main(String[] args) {
+		int arr[] = {2,5,9,7,4,6};
```

Comment:
Hi @ManojNAug19 
If I am passing 7 as number and check the output I am expecting it is existing number..
but showing me as missing number....
I think you not correct your logic yet...
please correct.....

check output for all the possible combination....